package.json

```diff
@@ -9,6 +9,7 @@
   },
   "lint-staged": {
     "**/*.{js,ts,jsx,tsx}": [
+      "./node_modules/.bin/eslint '**/*.{js,ts,jsx,tsx}' --fix",
```

Comment:
`npx eslint` will be shorter.

`npx` helps you to call any binary package in a short form. If the called package doesn't exist on your machine, `npx` will install it. If the package exists in the project – `npx` will use it. If not, it'll try to resolve global one.